### PR TITLE
[TASK][27192] Create models

### PR DESCRIPTION
**Pull request by kguen-sun**
https://edu-redmine.sun-asterisk.vn/issues/27192

### CODE REVIEW

**Review by nthtra-110273 — APPROVED** [review]

**Review by nguyenvd16 — APPROVED** [review]

**Comment by kguen-sun** [comment]
ready

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
+  get 'static_pages/home'
```
Comment (hieplt-0914) [inline config/routes.rb]: convention

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>StaticPages#home</h1>
```
Comment (hieplt-0914) [inline app/views/static_pages/home.html.erb]: Xóa nội dung cái này đi

**Review by nthtra-110273 — APPROVED** [review]

**Review by nguyenvd16 — APPROVED** [review]

**Review by nthtra-110273 — APPROVED** [review]

**Review by nthtra-110273 — APPROVED** [review]

**Review by nguyenvd16 — APPROVED** [review]

**Comment by kguen-sun** [comment]
ready

**Pull request merged**